File: spring/src/main/java/cn/foxkiar/loongarch/entity/Patrol.java
package cn.foxkiar.loongarch.entity;

import java.util.Date;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;

import lombok.Data;

@Data
public class Patrol {
    @TableId(type = IdType.INPUT)
    private Date recordDate;
    private Integer personId;
    private String personName;
    private Boolean isNormal;
    private String comment;

    @TableField(exist = false)
    private Long startDate;
    @TableField(exist = false)
    private Long endDate;
}

File: uniapp/src/api/patrol.ts
export class Patrol {
    recordDate: Date;
    userId: number;
    userName: string;
    isNormal: boolean;
    comment: string;
    startDate: Date;
    endDate: Date;
}

export class PatrolPage {
    records: Array<Patrol>;
    total: number;
    size: number;
    current: number;
}

export const patrolRules = {}